test/integration/image-component/default/pages/layout-responsive.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import Image from 'next/image'
+
+const Page = () => {
+  return (
+    <div>
+      <p>Layout Responsive</p>
+      <Image
+        id="responsive1"
+        src="/wide.png"
+        width="1200"
+        height="700"
+        layout="responsive"
+      ></Image>
+      <Image
+        id="responsive2"
+        src="/wide.png"
+        width="1200"
+        height="700"
+        layout="responsive"
+      ></Image>
+      <Image
+        id="responsive3"
+        src="/wide.png"
+        width="1200"
+        height="700"
+        layout="responsive"
+      ></Image>
+      <Image
+        id="responsive4"
+        src="/wide.png"
+        width="1200"
+        height="700"
+        layout="responsive"
+      ></Image>
```

Comment:
```suggestion
      ></Image>
                <Image
            id="responsive-unsized"
            src="/wide.png"
            unsized
            layout="responsive"
          />
```

I expected this to have the same behavior (100% width), but it doesn't (notice it's wider)
<img width="530" alt="Screen Shot 2020-10-29 at 3 18 50 PM" src="https://user-images.githubusercontent.com/34928425/97633653-0e750300-19fa-11eb-88e9-e9c8285490d1.png">